post/views.py

```
@@ -0,0 +1,417 @@
+                {"detail": "[title, content] fields missing."},
+                status=status.HTTP_400_BAD_REQUEST,
+            )
+        try:
+            author = User.objects.get(username=username)
+            if not author.check_password(password):
+                return Response(
+                    {"detail": "Password is incorrect."},
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+            post = Post.objects.create(title=title, content=content, author=author)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)